test/core/test/jest-expect.test.ts

```diff
@@ -332,6 +332,13 @@ describe('jest-expect', () => {
     expect(null).toBe(true)
   })
 
+  it.fails('toThrow should failure when does not throw ', () => {
+    expect(() => {
+    }).toThrow(Error)
+    expect(async () => {
```

Comment:
This line will never run. Either make 2 separate tests with `.fails` or wrap it inside another `expect(() => {}).toThrow()` and remove `.fails`